docs/services.py

```diff
@@ -89,17 +89,28 @@ async def register_service(callback):
         }
     )
 
+    # Create the annotator URL
     annotation_sid = svc["id"]
     config_str = f'{{"server_url": "{SERVER_URL}", "annotation_service_id": "{annotation_sid}", "token": "{token}"}}'
     encoded_config = urllib.parse.quote(
         config_str, safe="/", encoding=None, errors=None
     )
     annotator_url = f"https://imjoy.io/lite?plugin={PLUGIN_URL}&config={encoded_config}"
```

Comment:
Note that the annotation plugin need to updated to use hypha rpc instead of imjoy rpc. Also I would create this annotation url in js instead of python.